### PR TITLE
UA changes for mtls login support

### PR DESCRIPTION
**Pull request by EpsilonTal**
UA changes for mtls login support

### CODE REVIEW

**Review by pavelmaliy — APPROVED** [review]

File: docs/commands/login.md [hunk 1]
```diff
@@ -59,6 +59,51 @@ login, l
...
+    Options: <code>password</code> / <code>client-credentials</code> (default is <code>password</code> flow)
+  </p>
+</details>
+<details>
+  <summary>client id</summary>
+  <p>
+    <code>--client-id</code>
+  </p>
+  <p>
+    The technical client-id provided on the binding creation, used for the <code>client-credentials</code> flow.
```
Comment (EpsilonTal) [inline docs/commands/login.md:77]: For review

File: docs/commands/login.md [hunk 2]
```diff
@@ -59,6 +59,51 @@ login, l
...
+  </p>
+</details>
+<details>
+  <summary>client secret</summary>
+  <p>
+    <code>--client-secret</code>
+  </p>
+  <p>
+    The technical client-secret provided on the binding creation, used for the <code>client-credentials</code> flow.
```
Comment (EpsilonTal) [inline docs/commands/login.md:86]: UA review

File: docs/commands/login.md [hunk 3]
```diff
@@ -59,6 +59,51 @@ login, l
...
+  </p>
+</details>
+<details>
+  <summary>certificate</summary>
+  <p>
+    <code>--cert</code>
+  </p>
+  <p>
+    The path to the file which contains the <code>certificate</code> (public-key) provided on the binding creation, used for the <code>client-credentials</code> flow.
```
Comment (EpsilonTal) [inline docs/commands/login.md:95]: UA review

File: docs/commands/login.md [hunk 4]
```diff
@@ -59,6 +59,51 @@ login, l
...
+  </p>
+</details>
+<details>
+  <summary>private key</summary>
+  <p>
+    <code>--key</code>
+  </p>
+  <p>
+    The path to the file which contains the <code>key</code> (private-key) provided on the binding creation, used for the <code>client-credentials</code> flow.
```
Comment (EpsilonTal) [inline docs/commands/login.md:104]: UA review

**Review by EpsilonTal — COMMENTED** [review]
UA review

File: docs/commands/login.md [hunk 5]
```diff
@@ -59,6 +59,51 @@ login, l
     User ID.
   </p>
 </details>
+<details>
+  <summary>auth flow</summary>
+  <p>
+    <code>--auth-flow</code>
+  </p>
+  <p>
+    Options: <code>password</code> / <code>client-credentials</code> (default is <code>password</code> flow)
```
Comment (EpsilonTal) [inline docs/commands/login.md:68]: UA review

File: docs/commands/login.md (hunk 1 above)
Comment (i522006) [inline docs/commands/login.md:77]: The technical client ID that was provided during the creation of the binding and used for the <code>client-credentials</code> authorization flow.

File: docs/commands/login.md (hunk 2 above)
Comment (i522006) [inline docs/commands/login.md:86]: The technical client secret that was provided during the creation of the binding and used for the <code>client-credentials</code> authorization flow.

File: docs/commands/login.md (hunk 3 above)
Comment (i522006) [inline docs/commands/login.md:95]: A path to the file that contains the public key <code>certificate</code> that was provided during the creation of the binding, used for the <code>client-credentials</code> authorization flow. 
@EpsilonTal was the public key provided or generated when we created the binding?

File: docs/commands/login.md (hunk 4 above)
Comment (i522006) [inline docs/commands/login.md:104]: A path to the file that contains the private <code>key</code>  that was provided during the creation of the binding, used for the client-credentials authorization flow.
@EpsilonTal was the public key provided or generated when we created the binding?

**Review by i522006 — CHANGES_REQUESTED** [review]

File: docs/commands/login.md (hunk 1 above)
Comment (i522006) [reply docs/commands/login.md:77]: The technical client ID that was generated upon the creation of binding and that is used for the <code>client-credentials</code> authorization flow.

File: docs/commands/login.md (hunk 2 above)
Comment (i522006) [reply docs/commands/login.md:86]: The technical client secret that was generated upon the creation of binding and that is used for the <code>client-credentials</code> authorization flow.

File: docs/commands/login.md (hunk 3 above)
Comment (i522006) [reply docs/commands/login.md:95]: A path to the file that contains the public key <code>certificate</code> that was generated upon the creation of binding and that is used for the <code>client-credentials</code> authorization flow.

File: docs/commands/login.md (hunk 4 above)
Comment (i522006) [reply docs/commands/login.md:104]: A path to the file that contains the private <code>key</code> that was generated upon the creation of binding and that is used for the <code>client-credentials</code> authorization flow.

File: docs/commands/login.md (hunk 1 above)
Comment (EpsilonTal) [reply docs/commands/login.md:77]: done

File: docs/commands/login.md (hunk 2 above)
Comment (EpsilonTal) [reply docs/commands/login.md:86]: done

File: docs/commands/login.md (hunk 3 above)
Comment (EpsilonTal) [reply docs/commands/login.md:95]: done

File: docs/commands/login.md (hunk 4 above)
Comment (EpsilonTal) [reply docs/commands/login.md:104]: done

**Review by i522006 — APPROVED** [review]

**Pull request merged**